_dev/storybooks.js

```diff
@@ -40,7 +49,7 @@ const customizeWebpackConfig = ({ config, mode }) => {
     assetLoader = {
       loader: resolve(
         __dirname,
-        '../../node_modules/@storybook/core/node_modules/file-loader/dist/cjs.js'
+        '../node_modules/@storybook/core/node_modules/file-loader/dist/cjs.js'
```

Comment:
We should avoid relying on exact paths under `node_modules` since hoisting plus version changes can move them unexpectedly. Since this depends on `file-loader` we should add it as a dependency and use `require.resolve` here.

idk what I think about putting this in `_dev`. I think I understand why you did, but im leaning toward fxa-react being the better place since storybook and react go hand in hand.

Reply:
Yeah, same with `_dev/rescripts.js` - that's very specifically a Create React App thing. I'd keep that stuff in fxa-react

Reply:
Truuue, but it could be argued that Tailwind and Storybook are not necessarily exclusive to React (though they are in our case, currently). I put them here for the same reason the TypeScript configs live here, as common build configs (none of them are bundled). But I do take your point, they'll live in `fxa-react/configs` for now 😄